python/ray/rllib/a3c/a3c.py

```diff
@@ -73,9 +73,8 @@ def get_completed_rollout_metrics(self):
         return completed
 
     def start(self):
-        logdir = get_tensorflow_log_dir(self.logdir)
         summary_writer = tf.summary.FileWriter(
-            os.path.join(logdir, "agent_%d" % self.id))
```

Comment:
This won't work if the logdir is logging to S3, as unfortunately TensorFlow doesn't support the s3:// protocol (that's what the get_tensorflow_log_dir fix was for)